### ISSUE TITLE
Syntax error when using interpolation in properties

### ISSUE THREAD

**Issue by endofunky** [sass/sass#198]
SCSS raises syntax errors in certain cases when using interpolation in property names with a leading `-`:

``` SCSS
@mixin vendor-prefix($property, $vendors, $value) {
  @each $vendor in $vendors {
    @if $vendor == none {
      #{$property}: $value;
    } @else {
      // Without the leading #{""} the parser raises a syntax error:
      //
      //   -#{$vendor}-#{$property}: $value;
      //
      // => Invalid CSS after "-": expected number or function, was "#{$vendor}-#{$p..."
      //
      #{""}-#{$vendor}-#{$property}: $value;
    }
  }
}

#test {
  @include vendor-prefix(border-radius, moz webkit ms o none, 1px);
}
```

**Comment by sunny** [comment]
A syntax error also appears when the interpolation is at the _start_ of the _second_ argument when a property has arguments seperated by commas.

Test case:

``` scss
// Works
p { background: a,     #{foo}bar;    }
p { background:        #{foo}bar, b; }
p { background: a, spam#{foo}bar, b; }

// Fails
// ERROR: Sass > Syntax error: Invalid CSS after "...,     #{foo}bar": expected ";", was ", b; }"
//      on line 9 of test.scss
p { background: a,     #{foo}bar, b; }
```

(Sass 3.1.10)

**Comment by nex3** [comment]
Closed by d5ec0586cd5cf40c31c5d99abf740a3b83204b5d
